### ISSUE TITLE
[question] where is the pretrained model  of scrfd_10g_bnkps

### ISSUE THREAD

**Issue by ShiyangZhang** [deepinsight/insightface#2158]
hi, I want to regenerate scrfd_10g_bnkps.onnx. Where is the pth file? I can only get pth model of scrfd_10g_kps.Are they the same?

**Comment by HeChengHui** [comment]
https://drive.google.com/file/d/18wEUfMNohBJ4K3Ly5wpTejPfDzp-8fI8/view

I got mine through this link. Is in the antelopev2 folder. Though I am not sure what is the difference between bnkps and kps. bnkps also has the 5 points kps function.

**Comment by ShiyangZhang** [comment]
> 

thx. It seems that no batchnorm parameters exists in kps.
